### PR TITLE
Game doesn't start 'till player moves, player starting position is a thing.

### PR DESCRIPTION
**Pull request by skylarjhdownes**

### CODE REVIEW

File: source/scripts/models/Ninja.js [hunk 1]
```diff
@@ -67,22 +68,27 @@ Ninja.prototype.update = function(fluxdelta, delta) {
...
         } if(Keyboard.isDown("D")
         || Keyboard.isDown("<right>")) {
+            this.hasMoved = true
             this.x += this.speed * fluxdelta
             this.direction = -1
         }
         while(Mouse.events.length > 0) {
             var event = Mouse.events.shift()
             if(event.type == "click") {
+                this.hasMoved = true
```
Comment (ehgoodenough) [inline source/scripts/models/Ninja.js:136]: I bet this was a funny bug. :p

"I'll just stand here, not move, but throw ninjastars at all the enemies." :p

File: source/scripts/models/Ninja.js [hunk 2]
```diff
@@ -6,11 +6,11 @@ var NinjaStar = require("./NinjaStar")
...
 var Images = require("../data/Images.js")
 
-var Ninja = function() {
+var Ninja = function(protoninja) {
     window.game.ninja = this
 
-    this.x = WIDTH / 2
-    this.y = HEIGHT / 2
+    this.x = protoninja.x || protoninja.tx * 32 || WIDTH / 2
+    this.y = protoninja.y || protoninja.ty * 32 || HEIGHT / 2
```
Comment (ehgoodenough) [inline source/scripts/models/Ninja.js:14]: Niiiice. @eCorbeau will like this. :3

**Comment by skylarjhdownes** [comment]
The build for 1a456d0 is dead: http://yeahgoodenough.com/skylarjhdownes/papercut/c/1a456d0/index.html
So I've presumably screwed up. 

Shouldn't there be one for b35d797?  I don't see one in the server list.

**Comment by ehgoodenough** [comment]
Nah, the build server is messed up for some reason. Probably NVM again. :p

File: source/scripts/models/Ninja.js (hunk 1 above)
Comment (skylarjhdownes) [inline source/scripts/models/Ninja.js:136]: I did see that one coming before it happened, believe it or not.  Wouldn't surprise me if there's some other thing like that that I didn't notice though.

**Comment by skylarjhdownes** [comment]
Ah, kk.

**Comment by skylarjhdownes** [comment]
I really ought to have made a branch for these boss monster files.  The selfie thing appears to work now.
selfie-0

**Pull request merged**